### ISSUE TITLE
Failure running XeroxPrinterDriver recipe

### ISSUE THREAD

**Issue by micmil** [autopkg/n8felton-recipes#127]
No match found on URL: https://www.support.xerox.com/support/workcentre-6655/downloads/enus.html?operatingSystem=macOS10_14
Failed.

**Comment by n8felton** [comment]
Looks like Xerox moved around their support/downloads page again. I'll need to poke at updating the recipe.

Probably going to switch to using https://www.support.xerox.com/en-us/product/versalink-c7020-c7025-c7030/downloads?platform=macOS10_15&language=en as the default link

**Comment by MLBZ521** [comment]
I have a [Xerox Printer Driver recipe](https://github.com/autopkg/MLBZ521-recipes/tree/master/Xerox%20Print%20Drivers) as well that is working.

Xerox continuously changes their site and how things work...constant cat and mouse.

**Comment by flammable** [comment]
I'm not as great with creating AutoPkg recipes as I should be, but I managed to adapt @n8felton's Munki recipe for @MLBZ521's download recipe. Thank you both!

https://github.com/autopkg/flammable-recipes/blob/master/Xerox/XeroxPrintDriver.munki.recipe

Nate, if you're good with it, I'll submit a pull request to add a [DeprecationWarning](https://github.com/autopkg/autopkg/wiki/Processor-DeprecationWarning) for your Xerox recipes.

**Comment by n8felton** [comment]
I'm not sure that I want to deprecate my recipe(s)? I'd much rather accept a PR that fixes my recipe(s), rather than duplicating them in other repos.

**Comment by n8felton** [comment]
My download recipe is fixed again with fe43da45

**Comment by MLBZ521** [comment]
I believe I wrote my Xerox recipe well before my private repo was incorporated into the main autopkg org, so I believe I may have a few duplicates that I created for myself/my org that didn't exist in the autopkg org at the time.  I need to go through them and see what I can deprecate.  I also want to try and move some of my `jss` only recipes into the `jss-recipes` repo.

That said, I believe my Xerox Print Driver recipe is significantly different that yours Nate, so it's not a "like for like" duplicate.
